src/brain/operating_unit_recorder.cpp

```diff
@@ -160,6 +160,7 @@ void OperatingUnitRecorder::AggregateFeatures(brain::ExecutionOperatingUnitType
   // TODO(wz2): Populate actual num_rows/cardinality after #759
   size_t num_rows = 1;
   size_t cardinality = 1;
+  size_t num_iterations = 0;
```

Comment:
Call it `num_loops`. This would be similar to Postgres terminology and that's also the term I used in the paper.